src/Frontend/Themes/Bootstrap/Modules/Blog/Layout/Templates/Archive.html.twig

```diff
@@ -0,0 +1,56 @@
+{#
+  variables that are available:
+  - {{ archive }}: contains an array with some dates
+  - {{ items }}: contains an array with all items, each element contains data about the items
+#}
+
+{% if not items %}
+  <section id="blogArchive" class="blog">
+  <div class="bd content">
+    <p>{{ 'msg.BlogNoItems'|trans }}</p>
+  </div>
+  </section>
+{% endif %}
+{% if items %}
+  <section id="blogArchive" class="blog">
+  <header role="banner">
+    <h3>
+    {% if archive.month %}
+      {{ 'msg.ArticlesFor'|trans|capitalize|format(archive.start_date|date('F Y',LANGUAGE ) ) }}
+    {% endif %}
+    {% if not archive.month %}
+      {{ 'msg.ArticlesFor'|trans|capitalize|format(archive.start_date|date('Y' ) ) }}
+    {% endif %}
+    </h3>
+  </header>
+  <div class="bd content">
+    <table class="dataGrid table table-hover" width="100%" itemscope itemtype="http://schema.org/Blog">
+    <thead class="hide">
+      <tr>
+      <th class="date">{{ 'lbl.Date'|trans|capitalize }}</th>
+      <th class="title">{{ 'lbl.Title'|trans|capitalize }}</th>
+      {% if allowComments %}<th class="comments">{{ 'lbl.Comments'|trans|capitalize }}</th>{% endif %}
+      </tr>
+    </thead>
+    <tbody>
+      {% for item in items %}
+      <tr {% if item.first %}class="firstChild"{% endif %}>
+        <td class="date muted"><time itemprop="datePublished" datetime="{{ item.publish_on|date('Y-m-d\TH,i,s' ) }}">{{ item.publish_on|date(dateFormatShort,LANGUAGE ) }}</time></td>
+        <td class="title"><a href="{{ item.full_url }}" title="{{ item.title }}">{{ item.title }}</a></td>
+        {% if allowComments %}
+        <td class="comments">
+          {% if item.comments %}
+          <i class="icon-comment"></i>
+          {% if item.comments_multiple %}<a href="{{ item.full_url }}#{{ 'act.Comments }}">{{ 'msg.BlogNumberOfComments''|trans|format(item.comments_count ) }}</a>{% endif %}
+          {% if not item.comments_multiple %}<a href="{{ item.full_url }}#{{ act.Comments }}">{{ 'msg.BlogOneComment'|trans }}</a>{% endif %}
```

Comment:
ik vermoed dat ge dat allemaal via een find and replace gedaan hebt en er dus overal eerst `if` en dan `if not` staat?